queries.sql

```diff
@@ -88,3 +88,31 @@ GROUP BY species;
 SELECT species, AVG(escape_attempts) FROM animals
 WHERE date_of_birth BETWEEN 'January 1, 1990' AND 'December 31, 2000'
 GROUP BY species;
+
+
+SELECT * FROM animals
+JOIN owners ON owner_id = owners.id
+WHERE full_name = 'Melody Pond';
+
+SELECT * FROM animals
+JOIN species ON species_id = species.id
+WHERE species.name = 'Pokemon';
+
+SELECT full_name AS owner_name, animals.name AS animal_name FROM owners
+LEFT JOIN animals ON animals.owner_id = owners.id;
+
+
+SELECT species.name AS species, count(*)  FROM animals
+JOIN species ON species_id = species.id
+GROUP BY species.id;
+
+
+SELECT animals.name AS animal 
+FROM animals
+JOIN species ON species_id = species.id
+WHERE species.name = 'Digimon' AND owner_id = (SELECT id FROM owners WHERE full_name = 'Jennifer Orwell');
+
+SELECT animals.name AS animal 
+FROM animals
+JOIN owners ON owner_id = owners.id
+WHERE animals.escape_attempts = 0 AND owner_id = (SELECT id FROM owners WHERE full_name = 'Dean Winchester');
```

Comment:
- [ ] Good job adding the queries @ZuhRa-HashiMi, But I belive you missed a query to find `Who owns the most animals?`, Please consider adding this query.